api/src/main/java/com/epam/pipeline/manager/pipeline/PipelineManager.java

```diff
@@ -111,10 +112,10 @@ public Pipeline create(final PipelineVO pipelineVO) throws GitClientException {
             pipelineVO.setRepositoryType(RepositoryType.GITLAB);
         }
         if (StringUtils.isEmpty(pipelineVO.getRepository())) {
-            Assert.isTrue(RepositoryType.BITBUCKET != pipelineVO.getRepositoryType(),
-                    "Bitbucket repository creation supported from urls only");
-            Assert.isTrue(RepositoryType.BITBUCKET_CLOUD != pipelineVO.getRepositoryType(),
-                    "Bitbucket Cloud repository creation supported from urls only");
+            Assert.isTrue(RepositoryType.BITBUCKET != pipelineVO.getRepositoryType() &&
```

Comment:
Maybe it would be clearer to replace multiple expressions into one `RepositoryType.GITLAB.equals(pipeline.getRepositoryType()` as it done in `GitManager`?